test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app/ViewController.swift

```diff
@@ -150,5 +267,15 @@ class ViewController: UIViewController, WKUIDelegate, WKNavigationDelegate, UIDo
     override func viewDidLoad() {
         super.viewDidLoad()
         setupBackend()
+        if let standaloneFilename = configData["IMJS_STANDALONE_FILENAME"] as? String {
+            let documentsDirectory = getDocumentsDirectory()
+            let bimFile = documentsDirectory.appendingPathComponent(standaloneFilename)
+            setupFrontend(bimFile: bimFile)
+        }
+        else if let _ = authClient,
```

Comment:
Move to previous line with curly brace for formatting consistency.

Reply:
Fixed.